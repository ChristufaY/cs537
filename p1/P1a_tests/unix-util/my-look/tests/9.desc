stdin test: read from piped output
