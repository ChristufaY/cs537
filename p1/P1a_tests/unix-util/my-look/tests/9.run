cat ./tests/wordlist.txt | ./my-look kara
