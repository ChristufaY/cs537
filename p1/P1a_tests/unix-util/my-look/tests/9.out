karachi
karakalpak
karakoram
karat
karate
